build.gradle.kts

```
@@ -17,9 +17,12 @@ dependencies {
         exclude("com.google.auto.service", "auto-service-annotations")
     }
 
+    testImplementation("com.fasterxml.jackson.core:jackson-core:latest.release")
     testImplementation("com.fasterxml.jackson.core:jackson-databind:latest.release")
+    testImplementation("com.fasterxml.jackson.module:jackson-module-jaxb-annotations:latest.release")
     testImplementation("org.codehaus.jackson:jackson-core-asl:latest.release")
     testImplementation("org.codehaus.jackson:jackson-mapper-asl:latest.release")
+    testImplementation("org.codehaus.jackson:jackson-xc:latest.release")
```

Similarly never used it seems; when I remove these locally the tests still pass.
```suggestion
    testImplementation("com.fasterxml.jackson.core:jackson-databind:latest.release")
    testImplementation("org.codehaus.jackson:jackson-core-asl:latest.release")
    testImplementation("org.codehaus.jackson:jackson-mapper-asl:latest.release")
```